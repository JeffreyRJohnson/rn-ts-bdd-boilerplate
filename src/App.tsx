import * as React from 'react';
import { Component } from 'react';
import { View, Text } from 'react-native';

class App extends Component {
  render() {
    return (
      <View>
        <Text>Hello Clarice </Text>
      </View>
    );
  }
}

export default App;
