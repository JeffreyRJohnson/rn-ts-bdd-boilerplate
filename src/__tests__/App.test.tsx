import 'jest';
import * as React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';
import App, { AppState } from '../App';

describe('App.tsx', () => {
  const app = renderer.create(<App />);

  it('renders correctly', () => {
    expect(app).toMatchSnapshot();
  });
});
