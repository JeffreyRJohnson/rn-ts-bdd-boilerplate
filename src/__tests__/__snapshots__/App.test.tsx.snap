// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.tsx renders correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    Hello Clarice 
  </Text>
</View>
`;
